engine/gameobject/src/gameobject/gameobject.cpp

```diff
@@ -2464,6 +2465,20 @@ namespace dmGameObject
 
         bool ret = true;
 
+
+        UpdateContext dynamic_update_context;
+        dynamic_update_context = *update_context;
+
+        // pass it as unscaled time
+        if (update_context->m_TimeScale > 0.001f )
```

Comment:
Why 0.001f specifically?

Reply:
It's just a threshold value. A sufficiently low value, have to pick one since it cannot be zero.